client/state/selectors/get-atomic-hosting-sftp-user.js

```diff
@@ -0,0 +1,19 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import { get } from 'lodash';
+
+/**
+ * Returns the sftp user details for given site and user
+ *
+ * @param  {Object}  state   Global state tree
+ * @param  {Number}  siteId The ID of the site we're querying
+ * @param  {Number}  userId  The ID of the user we're querying
+ * @return {Object} SFTP user details for selected user
+ */
+export function getAtomicHostingSFTPUser( state, siteId, userId ) {
+	return get( state, [ 'atomicHosting', 'sites', siteId, 'sftpUsers', userId ], null );
```

Comment:
If we use `keyedReducer` as suggested above, this should change to:
```suggestion
	return get( state, [ 'atomicHosting', siteId, 'sftpUsers', userId ], null );
```

Reply:
Done.